docs/app/pages/css/css-sections/icons/ux-icons/ux-icons.component.html

```diff
@@ -1,5 +1,18 @@
-<p *ngIf="iconSetClass === 'hpe-icon'">To use an icon simply apply the <code>{{ iconSetClass }}</code>
-    class along with one of the following classes:</p>
+<div *ngIf="iconSetClass === 'hpe-icon'" class="row m-b-md hpe-icon-instructions">
+
+    <div class="col-lg-12 col-md-12 col-sm-12">
+
+        <p>To use an <code>{{ iconSetClass }}</code> import the HPE Icons CSS file into your project.
```

Comment:
I think we should mark this icon set as deprecated at this point as well, so that noone is establishing new hpe-icons in their app. Let's put up a warning before this paragraph indicating that `hpe-icons` is deprecated and will be removed in the next major release.